### DISCUSSION TITLE
Integration of new visualization techniques with python libraries to geonode dashboard widgets.

### DISCUSSION THREAD

**Discussion by romelvazquez** [GeoNode/geonode-project#537] [Ideas]
Hello, I would like to propose an article with my students for the FOSS4G Belem 2024 Congress, related to the integration of new visualization techniques with python libraries to geonode dashboard widgets. This could help decision makers. Classical visualization techniques such as scatter plots, box plots, even visualization of these graphs on vector geometries as in most GIS could be useful and an important contribution to an OSGEO project with as much acceptance as geonode. If the main developers of the geonode dashboards could explain to me the main entry points in the code to incorporate new visualization techniques with python libraries such as matplotlib, seaborn, among others, it would be of great help to save time and complete the work and present it on the Congress important dates. Greetings.

**Comment by kikislater** [comment]
Geonode uses mapstore now. It relies on openlayers 3 or leaflet and react JS librairies

As far as I know, not all features from mapstore are implemented in geonode but could be extended.

Widgets documentation: https://mapstore2-offtherailz.readthedocs.io/en/stable/user-guide/widgets/

https://youtube.com/watch?v=ybuivDWypdQ
